### PR TITLE
No fakes

### PR DESCRIPTION
**Pull request by nishp1**
https://app.asana.com/0/45511869812564/64233498710418

From https://github.com/percolate/eslint-config-percolate/pull/13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,11 +17,12 @@
     "url": "https://github.com/percolate/eslint-config-percolate/issues"
   },
   "devDependencies": {
-    "eslint": "1.1.0",
-    "eslint-plugin-react": "3.2.0"
+    "eslint": "1.10.3",
+    "eslint-plugin-react": "3.2.0",
+    "eslint-plugin-percolate": "file:rules"
```
Comment (maxkostow) [inline package.json]: Is this necessary?

Comment (nishp1) [inline package.json]: This defines a plugin named `percolate` and value of `file:rules` maps it to rules directory.

Comment (maxkostow) [inline package.json]: So it needs to be routed through the `package.json`? We can't put something like `@percolate/file:.rules` in the plugins array?

Comment (maxkostow) [inline package.json]: https://github.com/eslint/eslint/blob/756b2790bf54e9f2df9a93dcfd40b8153a814e6a/lib/config.js#L99 guess not

Comment (nishp1) [inline package.json]: Don't think so, I went through eslint's code and it resolves it like any
other node module, so ended up doing same as React
https://github.com/facebook/react/blob/master/package.json#L17

On Tue, Dec 8, 2015 at 11:37 AM, Max Kostow <notifications@github.com>
wrote:

> In package.json
> <https://github.com/percolate/eslint-config-percolate/pull/14#discussion_r46977548>
> :
>
> > @@ -17,11 +17,12 @@
> >      "url": "https://github.com/percolate/eslint-config-percolate/issues"
> >    },
> >    "devDependencies": {
> > -    "eslint": "1.1.0",
> > -    "eslint-plugin-react": "3.2.0"
> > +    "eslint": "1.10.3",
> > +    "eslint-plugin-react": "3.2.0",
> > +    "eslint-plugin-percolate": "file:rules"
>
> So it needs to be routed through the package.json? We can't put something
> like @percolate/file:.rules in the plugins array?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/percolate/eslint-config-percolate/pull/14/files#r46977548>
> .
>



-- 
Nish Patel
Frontend Engineer
Get inspired. See what's new on the Percolate blog: blog.percolate.com

Comment (nishp1) [inline package.json]: Yep, thats it.

On Tue, Dec 8, 2015 at 11:40 AM, Max Kostow <notifications@github.com>
wrote:

> In package.json
> <https://github.com/percolate/eslint-config-percolate/pull/14#discussion_r46977992>
> :
>
> > @@ -17,11 +17,12 @@
> >      "url": "https://github.com/percolate/eslint-config-percolate/issues"
> >    },
> >    "devDependencies": {
> > -    "eslint": "1.1.0",
> > -    "eslint-plugin-react": "3.2.0"
> > +    "eslint": "1.10.3",
> > +    "eslint-plugin-react": "3.2.0",
> > +    "eslint-plugin-percolate": "file:rules"
>
>
> https://github.com/eslint/eslint/blob/756b2790bf54e9f2df9a93dcfd40b8153a814e6a/lib/config.js#L99
> guess not
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/percolate/eslint-config-percolate/pull/14/files#r46977992>
> .
>



-- 
Nish Patel
Frontend Engineer
Get inspired. See what's new on the Percolate blog: blog.percolate.com

**Pull request merged**